packages/opencensus-instrumentation-http/test/test-http.ts

```diff
@@ -290,6 +303,60 @@ describe('HttpPlugin', () => {
       });
     });
 
+    it('should create a child span for GET requests with tag context', () => {
```

Comment:
I'm not sure if I understand these tests - why test spans? Same for other tests.